public/scripts/countryData.js

```diff
@@ -0,0 +1,90 @@
+(function(module) {
+  var origin = {};
```

Comment:
You might also consider making `countries` a property of `origin`, and then putting each of your country objects inside of that property. Deeply nested objects are totally fine! 

That would have consolidated everything into one object, which lessens the amount of global namespace necessary; i.e. You would access everything through the `origin` object in the global scope.